Tools/i18n/pygettext.py

```diff
@@ -124,6 +124,15 @@
     --width=columns
         Set width of output to columns.
 
+    --omit-header
+        Do not write header to file.
+
+        This is useful for testing purposes because it eliminates a source of
+        variance for generated .mo files.
+
+        Note: Using this option will lead to an error during compilation or other
```

Comment:
It may lead and may not. By default, `msgfmt.py` uses Latin1 which never fails in decoding. It still can fail in encoding if there were escaped characters out of the Latin1 range.

Reply:
```suggestion
        Note: Using this option may lead to an error during compilation or other
```
Is this sufficient in your opinion?